sim/shaman/talents.go

```diff
@@ -178,7 +180,9 @@ func (shaman *Shaman) applyElementalFocus() {
 				shaman.PseudoStats.SchoolDamageDealtMultiplier[stats.SchoolIndexNature] /= oathBonus
 				shaman.PseudoStats.SchoolDamageDealtMultiplier[stats.SchoolIndexFire] /= oathBonus
 				shaman.PseudoStats.SchoolDamageDealtMultiplier[stats.SchoolIndexFrost] /= oathBonus
-				shaman.Earthquake.DamageMultiplierAdditive -= 0.05
+				if shaman.Earthquake != nil {
```

Comment:
When you fix open Issues I'd advise to add the note in the PR or respective commit.
`Fixes: #ticketNumber` Github will automaticaly link commit and issue so they're visible and close the ticket on merge.